tezos/api/src/ffi.rs

```diff
@@ -133,6 +133,7 @@ pub struct TezosContextConfiguration {
     pub enable_testchain: bool,
     pub readonly: bool,
     pub sandbox_json_patch_context: Option<PatchContext>,
+    pub timing_db_path: Option<PathBuf>,
```

Comment:
@sebastiencs I see it is optional here, but not elsewhere, why the mismatch? (it being optional is what we want)

Reply:
@sebastiencs btw, I am thinking that something like "context_stats_db" is a better name than "timings_db", we will probably use it for more stuff than just timings.